api_spec.yaml

```diff
@@ -3246,9 +3246,451 @@ paths:
+          $ref: '#/components/responses/401'
+        
+        '402':
+          $ref: '#/components/responses/402'
+
+        '404':
+          $ref: '#/components/responses/404'
+        
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/tenant/config:
+    get:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Get a tenant config.
+
+        If `tenantId` is not specified, `defaultTenantId` will be used
+
+        Returns recipes with their enabled flag and recipe specific configs saved in core
+
+      operationId: tenantConfigGet
+      parameters:
+        - $ref: '#/components/parameters/multitenancyRid'
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+        - name: tenantId
+          in: query
+          required: false
+          schema:
+            $ref: '#/components/schemas/tenantId'
+      responses:
+        200:
+          description: Create or Update result
+          content:
+            application/json:
+              schema:
+                oneOf:
+                - type: object
+                  required:
+                    - status
+                    - emailPassword
+                    - thirdParty
+                    - passwordless
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusOK'
+                    emailPassword:
+                      type: object
+                      required:
+                        - enabled
+                      properties:
+                        enabled:
+                          type: boolean
+                    thirdParty:
+                      type: object
+                      required:
+                        - enabled
+                      properties:
+                        enabled:
+                          type: boolean
+                        providers:
+                          type: array
+                          items:
+                            $ref: '#/components/schemas/thirdPartyProviderConfig'
+                    passwordless:
+                      type: object
+                      required:
+                        - enabled
+                      properties:
+                        enabled:
+                          type: boolean
+                - type: object
+                  required:
+                    - status
+                  properties:
+                    status:
+                      $ref: '#/components/schemas/statusTenantDoesNotExistError'
+
+        '400':
+          $ref: '#/components/responses/400'
+        
+        '401':
+          $ref: '#/components/responses/401'
+        
+        '402':
+          $ref: '#/components/responses/402'
+
+        '404':
+          $ref: '#/components/responses/404'
+        
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/config/thirdparty/provider:
```

Comment:
i think you should remove the `/provider` postfix in all these thirdparty related APIs. It's not needed

Reply:
Done